src/modules/app/containers/RouterFragment.js

```diff
@@ -30,6 +34,9 @@ export class RouterFragment extends React.Component {
     return <Fragment forRoute='/'>
       {/* Routes */}
       <React.Fragment>
+        <Helmet>
+          <title>{cityConfig.title}</title>
```

Comment:
So we always have the same title? And I think in NEUBURG-17 you removed title from the cityConfig.

Reply:
I don't know if it's required, for me it's also okay if we leave it  like this.